casepro/pods/registry.py

```diff
@@ -0,0 +1,37 @@
+from django.apps import apps
+from django.conf import settings
+
+
+def get_class_from_app_label(label):
+    '''
+    Given the label of a pod, get the Pod class associated with it.
+    '''
+    appconfig = apps.get_app_config(label)
+    return appconfig.pod_class
+
+
+def load_pod(index, config):
+    '''
+    Given the index of the pod, and the config dictionary for the pod, this
+    returns the instance of that pod.
+    '''
+    config = config.copy()
+    app_label = config.pop('type')
```

Comment:
Addressed in https://github.com/praekelt/casepro/pull/31/commits/bcc8013c2587fb612ee6a24b12c8572b95c2fa76